src/cockpit/bridge.py

```diff
@@ -192,6 +194,8 @@ def setup_logging(debug: bool):
 
 
 def main() -> None:
+    polyfills.install()
```

Comment:
Now that we know that {send,recv}_fds() are the only missing API that we still need, we actually could move them to bridge.py and transports.py, or call them as `polyfills.{send,recv}_fds()` Then we would not clutter global namespace. Let me know if you prefer that, I don't have a strong opinion either way.